classes/OccurrenceIndividual.php

```diff
@@ -962,41 +1040,44 @@ public function getDatasetArr(){
 		}
 		else $this->errorMessage = 'ERROR: Unable to set datasets for user: '.$this->conn->error;
 
-		$sql3 = 'SELECT datasetid, notes FROM omoccurdatasetlink WHERE occid = '.$this->occid;
-		$rs3 = $this->conn->query($sql3);
-		if($rs3){
-			while($r3 = $rs3->fetch_object()){
-				if(isset($retArr[$r3->datasetid])){
-					//Only display datasets linked to current user, at least for now. Once isPublic option is activated, we'll open this up further.
-					$retArr[$r3->datasetid]['linked'] = 1;
-					if($r3->notes) $retArr[$r3->datasetid]['notes'] = $r3->notes;
+		$sql3 = 'SELECT datasetid, notes FROM omoccurdatasetlink WHERE occid = ?';
+		if($stmt = $this->conn->prepare($sql3)){
+			$stmt->bind_param('i', $this->occid);
+			$stmt->execute();
+			if($rs3 = $stmt->get_result()){
+				while($r3 = $rs3->fetch_object()){
```

Comment:
All of the other ones looks like they're $rs. Why is this one r3? Not that it matters, but it broke the pattern.